c3po/api/service/paginate.py

```diff
@@ -0,0 +1,30 @@
+from collections import OrderedDict
+from urllib.parse import parse_qs, urlencode, urlparse, urlunparse
+
+
+def get_paginated_response(posts, url, total, start=0, limit=24):
+    response = {}
+    if limit > total:
+        return None
```

Comment:
Is this condition really needed? If limit=24 and total=10, we should return the 10, no?

Reply:
Done :+1: